src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/Ecma335/Encoding/ControlFlowBuilder.cs

```diff
@@ -75,14 +86,14 @@ internal int GetBranchDistance(ImmutableArray<int>.Builder labels, ILOpCode bran
             }
         }
 
-        private readonly ImmutableArray<BranchInfo>.Builder _branches;
```

Comment:
Why change to standard `List<>`? (affects allocs)

I assume there were too many late-bound adds?

Reply:
It just felt weird using an immutable collection builder with no intent to create an immutable collection. Is there any reason to prefer them over `List<>`?